spec/models/card_spec.rb

```diff
@@ -77,49 +77,49 @@
   it 'check_translation Eng OK' do
     card = Card.create(original_text: 'дом', translated_text: 'house',
                        user_id: 1, block_id: 1)
-    expect(card.check_translation('house')).to be 0
+    expect(card.check_translation('house')).to be true
   end
 
   it 'check_translation Eng NOT' do
     card = Card.create(original_text: 'дом', translated_text: 'house',
                        user_id: 1, block_id: 1)
-    expect(card.check_translation('RoR')).to be nil
+    expect(card.check_translation('RoR')).to be false
   end
 
   it 'check_translation Rus OK' do
     card = Card.create(original_text: 'house', translated_text: 'дом',
                        user_id: 1, block_id: 1)
-    expect(card.check_translation('дом')).to be 0
+    expect(card.check_translation('дом')).to be true
   end
 
   it 'check_translation Rus NOT' do
     card = Card.create(original_text: 'house', translated_text: 'дом',
                        user_id: 1, block_id: 1)
-    expect(card.check_translation('RoR')).to be nil
+    expect(card.check_translation('RoR')).to be false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.